### PR TITLE
FOUR-15851: On Task Completed - Default Behavior (front-end changes)

### PR DESCRIPTION
**Pull request by rodriquelca**
## Issue & Reproduction Steps
Issue when a user opens two modelers in the same browser

## Solution
- from-end element destination component
## How to Test
1. Open the modeler
2. Select a task
3. Configure element destination properties

## Related Tickets & Packages
- https://github.com/ProcessMaker/processmaker-bpmn-moddle/pull/58
- https://processmaker.atlassian.net/browse/FOUR-15851
- https://processmaker.atlassian.net/browse/FOUR-15854

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next

### CODE REVIEW

File: src/components/inspectors/ElementDestination.vue [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      default: '',
+    },
+  },
+  name: 'ElementDestination',
+  
+  data() {
+    return {
+      loading: false ,
+      validation: '',
+      destinationType: null,
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:64]: Here is a typo
```suggestion
      customDashboard: null,
```

File: src/components/inspectors/ElementDestination.vue [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        { value: 'second', title: 'Dashboard 2' },
+      ],
+      externalURL: '',
+    };  
+  },
+  watch: {
+    customDashboad() {
+      this.setBpmnValues({
+        title: this.customDashboad.title,
+        url: this.customDashboad.url,
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:92]: Avoid using this in watch properties
```suggestion
    customDashboad(newValue) {
      this.setBpmnValues({
        title: newValue.title,
        url: newValue.url,
      });
    },
```

File: src/components/inspectors/ElementDestination.vue [hunk 3]
```diff
@@ -0,0 +1,204 @@
+<template>
+  <div>
+    <b-form-group :label="$t('Element Destination')">
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:3]: Missing translation

File: src/components/inspectors/ElementDestination.vue [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+      <b-form-select id="" v-model="destinationType" @change="destinationTypeChange" data-test="element-destination-type">
+        <b-form-select-option :value="null" disabled>{{ $t('Select element destination') }}</b-form-select-option>
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:5]: Missing translation

File: src/components/inspectors/ElementDestination.vue [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+        <option v-for="option in options" :key="option.value" :value="option.value">{{ $t(option.content) }}</option>
+      </b-form-select>
+      <small class="form-text text-muted">{{ $t("Enter the destination...") }}</small>
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:8]: Missing translation

File: src/components/inspectors/ElementDestination.vue [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+    </b-form-group>
+
+    <form-multi-select
+      v-if="destinationType === 'customDashboard'"
+      :label="$t('Dashboard')"
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:13]: Missing translation

File: src/components/inspectors/ElementDestination.vue [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+      name="Dashboard"
+      :helper="$t('Select the dashboard to show the summary of this request when it completes')"
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:15]: Missing translation

File: src/components/inspectors/ElementDestination.vue [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+      @search-change="searchChange"
+      data-test="dashboard"
+    />
+    <form-input
+      v-if="destinationType === 'externalURL'" 
+      :label="$t('URL')"
+      v-model="externalURL"
+      :error="getValidationErrorForURL(externalURL)"
+      data-cy="events-add-id"
+      :placeholder="urlPlaceholder"
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:39]: Missing translation and here is a typo
```suggestion
      :helper="$t('Determine the URL where the request will end')"
```

File: src/components/inspectors/ElementDestination.vue [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+      this.loadDashboards();
+    }
+  },
+  mounted() {
+    this.urlModel = { ...this.defaultValues };
+    this.loadData();
+    
+  },
+  methods: {
+    getValidationErrorForURL(url) {
```
Comment (devmiguelangel) [inline src/components/inspectors/ElementDestination.vue:119]: Missing translation

**Review by devmiguelangel — CHANGES_REQUESTED** [review]

File: src/components/inspectors/ElementDestination.vue (hunk 4 above)
Comment (rodriquelca) [reply src/components/inspectors/ElementDestination.vue:5]: added to https://github.com/ProcessMaker/processmaker/pull/6875/files

File: src/components/inspectors/ElementDestination.vue (hunk 1 above)
Comment (rodriquelca) [reply src/components/inspectors/ElementDestination.vue:64]: fixded

File: src/components/inspectors/ElementDestination.vue (hunk 9 above)
Comment (rodriquelca) [reply src/components/inspectors/ElementDestination.vue:119]: added to https://github.com/ProcessMaker/processmaker/pull/6875/files

File: src/components/inspectors/ElementDestination.vue (hunk 2 above)
Comment (rodriquelca) [reply src/components/inspectors/ElementDestination.vue]: fixed

**Review by devmiguelangel — APPROVED** [review]

**Pull request merged**